docs/simplesamlphp-upgrade-notes-2.0.md

```diff
@@ -144,3 +145,53 @@ $x = \SimpleSAML\Utils\Arrays::arrayize($someVar)
 
 [1]: https://github.com/simplesamlphp/simplesamlphp/wiki/Migrating-translations-(pre-migration)
 [2]: https://github.com/simplesamlphp/simplesamlphp/wiki/Twig:-Migrating-templates
+
+## Upgrading and EntityIDs
+
+If you still have your 1.x installation available, the entityID you
+are using for your SP and IdP should be available in
+module.php/core/frontpage_federation.php location on your
+SimpleSAMLphp server.
+
+An EntityID is a unique identifier for your SP or IdP. It has to be
+unique, like an IP-address has to be unique in your network. You might
+like to use your host name and some additional path information for
+the EntityID.
+
+The EntityID is set in two locations, as the property 'entityID' for
+an SP and as the index in the $metadata array for an IdP. Examples of
+both are shown below.
+
+For the SP you can set the EntityID as shown in the below fragment of
+authsources.php. In all of the below configuration fragments the
+EntityID is set to (<https://example.com/the-service/>).
+
+```php
+...
+    'default-sp' => [
+        'saml:SP',
+        // The entity ID of this SP.
+        'entityID' => 'https://example.com/the-service/',
+...
+```
+
+One suggestion for forming an EntityID is to use the below scheme.
+
+```php
+$entityid_sp = 'https://'
+   . $_SERVER['HTTP_HOST']
+   . '/simplesaml/module.php/saml/sp/metadata.php/default-sp';
+```
+
+For an IdP you might like to look at saml20-idp-hosted.php where the
+EntityID is used as the key in the metadata array.
+
+```php
+...
+$metadata['https://example.com/the-service/'] = [
+...
+```
+
+If you are using Azure and if you are configuring a hosted sp in ssp,
+the entityID will be a relying party you added in azure. It will have
+an entityid in azure like <https://your.ssp.hostname/default-sp>.
```

Comment:
This last paragraph seems a bit out of place. I assume it's about using Azure (now called Entra) as an IdP? So this could genericallt be said as "If you use SimpleSAMLphp as an SP, the IdP you are using will have your correct entityID confifgured"? There seems nothing special about Azure (Entra) in this regard.

Reply:
ok, I will change the last paragraph to the suggested sentence.